internal/dumper/loader.go

```diff
@@ -345,19 +441,47 @@ func (l *Loader) restoreTable(ctx context.Context, table string, conn *Connectio
 		return 0, err
 	}
 	query1 := string(data)
-	querys := strings.Split(query1, ";\n")
+	queries := strings.Split(query1, ";\n")
```

Comment:
Similar here, we should split with the parser instead.

Reply:
Incorporated these adjustments into the most recent commit, thank you @dbussink!